Sources/Core/Presenter/UploadingItem.swift

```diff
@@ -118,12 +119,13 @@ public final class UploadingItem: Equatable {
         self.attachment = attachment
         extraData = nil
     }
-    
+
     /// Init an uploader item with a given uploaded file.
     ///
     /// - Parameters:
     ///   - attachment: an uploaded file attachment.
     ///   - fileName: an uploaded file name.
+    @available(*, deprecated, message: "Please use `init(channel:url:)` initializer")
     public init(attachment: Attachment, fileName: String) {
```

Comment:
@b-onc why you deprecate this and method above? :)
It should be used for items that were uploaded, e.g. reused in collection cells.

Reply:
But they're not used? At least I couldn't find where they're used. Can you point me?

Reply:
You're right. I can't find them in usage. My example was wrong. I think it was supposed to be used for editing messages where you already have uploaded attachments and can delete them, but it wasn't implemented. On edit, users can only edit text or remove a message completely.

Reply:
Thank you for explaining! 🙌